actions/global_actions.jsx

```diff
@@ -61,21 +58,13 @@ export function emitChannelClickEvent(channel) {
     }
     function switchToChannel(chan) {
         const state = getState();
-        const getMyChannelMemberPromise = dispatch(getMyChannelMember(chan.id));
-        const oldChannelId = getCurrentChannelId(state);
         const userId = getCurrentUserId(state);
         const teamId = chan.team_id || getCurrentTeamId(state);
         const isRHSOpened = getIsRhsOpen(state);
         const isPinnedPostsShowing = getRhsState(state) === RHSStates.PIN;
-        const member = selectMyChannelMember(state, chan.id);
+        const member = getMyChannelMember(state, chan.id);
 
-        getMyChannelMemberPromise.then(() => {
-            dispatch(getChannelStats(chan.id));
-
-            // Mark previous and next channel as read
-            dispatch(markChannelAsRead(chan.id, oldChannelId));
-            dispatch(markChannelAsViewed(chan.id, oldChannelId));
-        });
+        dispatch(getChannelStats(chan.id));
```

Comment:
This was done few years back here and i cannot get the context of this https://github.com/mattermost/mattermost-webapp/commit/031cf13df86eea16d5c5bc9bee6b74ee95d4e301. As of now we do fetch the members on socket reconnect. I don't see how we would need that ATM